### PR TITLE
Copy docs from the main project and redo the index

### PR DESCRIPTION
**Pull request by varyonic**
Follow on from resque/resque#1778 in response to resque/resque#1771

Preview currently at https://varyonic.github.io/resque.github.com/ 

I'm not sure which content from the wiki you wanted to include.  I did go through the plugins page and tried to organize.

Shall I move more of the README Additional Information?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,25 +0,0 @@
----
-layout: default
----
-
-<div class="home">
-
-  <h1 class="page-heading">Posts</h1>
-  
-  <a class='githublink' href="https://github.com/resque/resque">Source code and documentation on github</a>
```
Comment (iloveitaly) [inline index.html:9]: @varyonic we should keep the github link on the page.

File: index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Welcome
+
```
Comment (iloveitaly) [inline index.md:2]: I think a nice intro here would be helpful

```suggestion

Resque is a Redis-backed ruby library for creating background jobs, placing those jobs on multiple queues, and processing them later. Unlike [Sidekiq](https://sidekiq.org) (an well-designed & well-maintained alternative!) it forks a new process for each job, which makes it resilient to memory leaks and eliminates thread-safety concerns. It's been battle-tested with high production loads and has a robust ecosystem of plugins to fill various needs.

```

**Review by iloveitaly — CHANGES_REQUESTED** [review]
Great change! Could we add that GH link back in? Otherwise, this looks great.

Next step would be pulling in the plugin wiki, and other useful wiki pages.

Also would be great if we could use a better theme, but that's definitely low-pri.

Thanks for the help here

File: index.html (hunk 1 above)
Comment (varyonic) [reply index.html:9]: Added back 'View on Github', I'll leave styling to someone else.

I'm a little reluctant to pull in the plugins page.  I understand the wiki does not get indexed by Google but it has minimal barriers to contribution so anyone in the community can register their forks and add notes on older gems (eg. @pboling).  I do think the docsite should link to other gems adopted by the organization, ie. resque-scheduler (will resque-loner be revived?)  Which other plugins might be adopted?

Comment (iloveitaly) [reply index.html:9]: > but it has minimal barriers to contribution so anyone in the community can register their forks and add notes on older gems 

Yeah, this is a downside. I wonder if there's some GH tooling out there to sync GH wiki pages to a public website or something? Would be something useful to look into.

What gems/projects do this well? None really come to mind for me.

Comment (pboling) [reply index.html:9]: Will `resque-loner` be revived?  Unlikely, I think.  However, the lineage is alive (and yet withering, as I don't currently use Resque).
My `resque-unique_in_queue` was a rewrite of `resque-solo`, which was a rewrite of `resque-loner`. :)

Comment (varyonic) [reply index.html:9]: Right. I'm currently switching from resque-loner to resque-unique-at-runtime, but given that resque-loner has already been adopted by the resque organization perhaps PRs could be accepted for a v2.0?  If not then may as well archive it.

**Pull request merged**